app/templates/components/questionnaire/exomeseq-studytype-choice.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="col-md-12">
```

Comment:
Usually you could move this className declaration into exomeseq-studytype-choice.js and remove the extra div. Is it needed in this instance for some reason?

Reply:
It's not really needed, and it looks like both of the components derived from AnswerableField (StringField and ExomeseqStudytypeChoice) render the same `<div class="col-md-12">`.

I chose to keep this in the `.hbs` rather than the `.js` since it's exclusively presentation, and didn't think that setting a class `col-md-12` would be appropriate for all `AnswerableField`s, even if it makes sense for the two we have now.